views/partials/recentPublishes.handlebars

```diff
@@ -0,0 +1,59 @@
+<div class="panel">
+	<h2>My Publishes</h2>
+	<div class="row" id="recent-publishes"></div>
+</div>
+
+<script type="text/javascript" >
+    const publishDisplayArea = document.getElementById('recent-publishes');
+    let myPublishes = null;
+    function addToPublishList(publishData, key, publishList){
+        let listElement = document.createElement('li');
+        listElement.setAttribute('id', key);
+        let link = document.createElement('a');
+        link.setAttribute('href', `/${publishData.claimId}/${publishData.name}`);
+        link.setAttribute('class', 'link--primary');
```

Comment:
Does this have to be done this way? Looks like another benefit of storing this server side and using a token -- we can use our actual templating engine!